CHANGELOG.md

```diff
@@ -2,14 +2,16 @@
 
 #### General
 * [Enhancement] Add rpm output to build process
-* [Enhancement] Add support for RHEL 8.x and Ubuntu 22.10
+* [Enhancement] Add support for RHEL 8.x, Ubuntu 22.10, SLES 15, Amazon Linux 2023
 * [Enhancement] Add support for Python 3.7 to 3.10+
 
 #### Framework
 
 #### Modules
 * [Misc] Create Ubuntu specific copies of bcc modules to reflect the Debian package adding -bpfcc to the binary file names.
 * [New Module] Add enadiag net diagnostic module.
+* [New Module] Add sosreport os gather module.
```

Comment:
Belongs in pull [91](https://github.com/awslabs/aws-ec2rescue-linux/pull/91) else this is dependent upon it.

Reply:
Fixed